homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+    }  
+  }
+  .about{
+    &-items{
+      flex-direction: row;
+      align-items: end;
+    }
+    .about-img{
+      max-width: 350px;
+    }
+    &-item-right{
+      order: 2;
+      margin-top: 90px;
+      margin-left: 61px;
+    }
+    &-item-left{
+      order: 1;
+    }
+  }
+  .essensials{
+    &-title{
+      max-width: 785px;
+    }
+    &-cards{
+      margin-top: 46px;
+    }
+  }
+  .essensials-card{
+    display: flex;
+    justify-content: center;
+  }
+  .card{
+    margin-top: 30px;
+    &-second{
+      margin-left: 30px;
+    }
+    &-btn{
+      max-width: 230px;
+    }
+  }
+  .slider{
+    .slide-title{
+      max-width: 451px;
+      margin: 20px auto 0;
+      font-size: 18px;
+    }
+    &-text{
+      max-width: 451px;
+      margin: 20px auto 0;
+      font-size: 18px;
+    }
+  }
+  .slick-slide img{
+    max-width: 470px;
+    width: 100%;
+    margin: 20px auto 0;
+  }
+  .services-item-text{
+    padding: 0 100px;
+  }
+  .planse-card{
+    padding: 35px 5px 25px;
+  }
+  .card-item-wrapper::before{
+    width: 140px;
+  }
+  .card-item{
+    font-size: 10px;
+  }
+  .description{
+    max-width: 700px;
+  }
+}
+@include desktop{
+  .container{
+    max-width: 1200px;
+    width: 100%;
+    padding: 0 84px;
+  }
+  .btn{
+    max-width: 200px;
+    width: 100%;
+    font-size: 12px;
+  }
+  .title{
+    font-size: 42px;
+    line-height: 54px;
+  }
+  .text{
+    font-size: 13px;
+    line-height: 24px;
+  }
+  .hamburger-menu{
+    display: none;
+  }
+  .header{   
+  .header-button{
+      flex-direction: row;
+      max-width: 290px;
+      margin: 0 0 0 30px;
+      top: 0;
+      & .btn{
+        color: #fff;
+        border: 1px solid #fff;
+      }
+      & .get-btn{
+       margin-right: 20px;     
+      }
+      & .log-btn{
+        margin-top: 0;
+      }
+    }
+  }
+  .header-menu{
+    flex: 1;
+  }
+  .menu-body{
+    position: static;
+    transform: translateY(0px);
+    background: transparent;   
+    .menu-list{
+      position: relative;
+      flex-direction: row;
+      justify-content: flex-end;
+      width: 100%;       
+      padding-top: 0;
+      align-items: center;
+      .menu-item{
+        position: relative;
+        padding: 10px 10px;
+        .menu-link{
+          color: #fff;
+          &::after {     
+            background-color: #fff;
+          }
+          &.active{
+            transform: translateY(0px);
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+          }
+        }
+      }
+    } 
+  }  
+  .hero{
+    padding-bottom: 100px;
+    .container{
+      padding: 0 200px;
+    }
+    &-inner{
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 120px;
+    }
+    &-title{
+      font-size: 110px;
+      line-height: 130px;
+      letter-spacing: 2px;
+      padding: 160px 0 0;
+      max-width: 1110px;
+      padding-bottom: 0;
+    }
+    &-text{
```

Comment:
Don't forget to add spaces before `{` everywhere in css.

Reply:
done

Reply:
done